src/blocks/product-card.css

```diff
@@ -0,0 +1,109 @@
+/* #region Product Card */
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+  padding: 0 16px;
+  width: 200px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  align-self: center;
+  height: 134px;
+  width: 160px;
+  margin: 32px 0;
+}
+
+.product-card__name {
+  margin: 8px 0;
+}
+
+.product-card__device-model {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.product-card__code {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin: 8px 0;
+}
+
+.product-card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+
+  color: var(--main-accent);
+}
+
+.product-card__price-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.product-card__price-label {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.product-card__price {
+  margin: 16px 0;
+
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  text-align: right;
+  color: var(--main-accent);
+}
+
+.product-card__buy-link {
+  cursor: pointer;
+
+  color: white;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  text-decoration: none;
+
+  display: inline-block;
+  padding: 12px 70px;
+  margin-bottom: 16px;
+
+  border: 1px solid var(--blue-accent);
+  background: var(--blue-accent);
+  border-radius: 5px;
+
+  transition: all 0.3s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated